debug_toolbar/forms.py

```diff
@@ -47,7 +47,6 @@ def verified_data(self):
 
     @classmethod
     def sign(cls, data):
-        items = sorted(data.items(), key=lambda item: item[0])
         return signing.Signer(salt=cls.salt).sign(
-            json.dumps({key: force_str(value) for key, value in items})
+            json.dumps({key: force_str(value) for key, value in data.items()})
```

Comment:
I was a bit worried about this. I thought that maybe the string was hashed and that changing the order of keys may lead to invalid hashes but this doesn't seem to be the case since we're not hashing, only signing and unsigning.